src/components/MyProfile.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import MissionProfile from './missionspage/missionsProfile';
+import RocketProfile from './rocketsComponent/RocketProfile';
+
+const MyProfile = () => (
+
+  <main className="profile-main">
+    <MissionProfile />
+    <RocketProfile />
```

Comment:
- [ ] It's better to use Routers the just add the components.

![Capture web_27-1-2022_121011_github com](https://user-images.githubusercontent.com/61562626/151347807-bac648bb-61e0-4416-8780-82caed112800.jpeg)

Reply:
We appreciate your review @dasileker  but recording a video is not part of the project requirements so we would really appreciate if you rethink that part of the review for it  is an optional thing. thanks

Reply:
@dasileker  please shade more light on the use Routers topic whats wrong with the way we did it ?? Isn't the application functional.??? We implemented all the requirements so why is our case different ?? We would really appreciate a quick response from you on this matter. Thanks

Reply:
@KayLemba the video is necessary because it's a capstone project and like every capstone project  you should add a video with your partner to show the specific of your codes

- the Router is like you did in your Calculator project instead of calling the components, you can add Router [see here](https://reactrouterdotcom.fly.dev/docs/en/v6/getting-started/tutorial)

Reply:
@dasileker 
Helllo there, 
Capstone project are we built at the end of module.. this project is for 4th week of react-module.
Capstone project we supposed to build in next week.
Please read this thread: https://github.com/microverseinc/curriculum-html-css/blob/main/articles/capstone_intro.md